WordPress/src/main/java/org/wordpress/android/ui/prefs/WPSwitchPreference.java

```diff
@@ -40,6 +39,10 @@ protected void onBindView(@NonNull View view) {
             Resources res = getContext().getResources();
             titleView.setTextSize(TypedValue.COMPLEX_UNIT_PX, res.getDimensionPixelSize(R.dimen.text_sz_large));
             titleView.setTextColor(res.getColor(isEnabled() ? R.color.grey_dark : R.color.grey_lighten_10));
+
+            if (!TextUtils.isEmpty(getDependency())) {
```

Comment:
It might be good to add a comment to point out when this will get triggered. If it's too obvious please feel free to skip it.